src/components/search/LegalSearch.tsx

```
@@ -34,12 +25,8 @@ export function LegalSearch() {
         }
         supabase.from("legal_document").select("*")
             .ilike("body", `%${searchQuery}%`)
-            .then(({ data, error }) => {
-                if (error) {
-                    console.log("Error fetching documents:", error)
-                } else {
-                    setDocs(data as any)
-                }
+            .then(({ data }) => {
+                setDocs(data as any)
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)